react/example/app.js

```diff
@@ -1,23 +1,34 @@
-import React from 'react'
-import MaskedInput from '../src/reactTextMask'
+import React from 'react';
+import MaskedInput from '../src/reactTextMask';
 
-export default React.createClass({
-  render() {
-    return (
-      <form className='form-horizontal'>
-        <div className='form-group'>
-          <label htmlFor='1' className='col-sm-2 control-label'>Masked input</label>
+export default () => (
```

Comment:
does the example app need these changes?

Are the changes this PR makes backwards compatible?  will existing users of the react component have to make any changes to their app/implementation for this to work?